Gemfile

```diff
@@ -3,35 +3,17 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.0'
-# Use postgresql as the database for Active Record
-gem 'pg', '>= 0.18', '< 2.0'
-# Use Puma as the app server
-gem 'puma', '~> 3.11'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.5'
-# Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use ActiveStorage variant
-# gem 'mini_magick', '~> 4.8'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
 
-# Reduces boot times through caching; required in config/boot.rb
+gem 'aasm'
```

Comment:
Was there a particular reason you reorganised the `Gemfile` removing the sectioned comments here?

Reply:
Oh, no particular reason, I was just trying to clean it up a bit alphabetically so it's easy to navigate and I removed all the commented out gems + the comments.  I've brought back the sectioned comments now.